### PR TITLE
Improved realization modules status values

### PR DESCRIPTION
**Pull request by konadrian**
Improved realization modules status values

### CODE REVIEW

File: modules/Settings/Picklist/models/Module.php [hunk 1]
```diff
@@ -197,6 +232,18 @@ public function addDescriptionColumn(string $tableName)
...
+	/**
+	 * Add automation column to picklist.
+	 *
+	 * @param string $tableName
+	 *
+	 * @return bool
+	 */
+	public function addAutomationColumn(string $tableName)
+	{
+		return App\Db::getInstance()->createCommand()->addColumn($tableName, 'automation', 'tinyint(1)')->execute();
```
Comment (rskrzypczak) [inline modules/Settings/Picklist/models/Module.php]: ?

File: modules/Settings/Picklist/models/Module.php [hunk 2]
```diff
@@ -424,4 +471,39 @@ public function clearPicklistCache(string $fieldName, string $moduleName)
 		\App\Cache::delete('getCloseStatesByName', \App\Module::getModuleId($moduleName));
 		\App\Cache::delete('getCloseStates', \App\Module::getModuleId($moduleName));
 	}
+
+	/**
+	 * Get all automation statuses.
+	 *
+	 * @return int[]
+	 */
+	public static function getAutomationStatus(): array
```
Comment (rskrzypczak) [inline modules/Settings/Picklist/models/Module.php]:
```suggestion
	public static function getAutomationStatuses(): array
```

File: modules/Settings/Picklist/models/Module.php [hunk 3]
```diff
@@ -424,4 +471,39 @@ public function clearPicklistCache(string $fieldName, string $moduleName)
...
+	{
+		return [self::AUTOMATION_NO_CONCERN, self::AUTOMATION_OPEN, self::AUTOMATION_CLOSED];
```
Comment (rskrzypczak) [inline modules/Settings/Picklist/models/Module.php]:
```suggestion
		return [self::AUTOMATION_NO_CONCERN => 'LBL_AUTOMATION_NO_CONCERN', self::AUTOMATION_OPE => 'LBL_AUTOMATION_OPEN'N, self::AUTOMATION_CLOSED => 'LBL_AUTOMATION_CLOSED'];
```

File: modules/Settings/Picklist/models/Module.php [hunk 4]
```diff
@@ -424,4 +471,39 @@ public function clearPicklistCache(string $fieldName, string $moduleName)
...
+	}
+
+	/**
+	 * Get translation for automation status.
+	 *
+	 * @param int $value
+	 *
+	 * @return string
+	 */
+	public static function getAutomationTranslation(int $value): string
```
Comment (rskrzypczak) [inline modules/Settings/Picklist/models/Module.php]: remove

File: modules/Settings/Picklist/models/Module.php [hunk 5]
```diff
@@ -83,6 +104,12 @@ public function addPickListValues($fieldModel, $newValue, $rolesSelected = [], $
 			}
 			$row['description'] = $description;
 		}
+		if (!empty($automation)) {
+			if (!$this->checkColumn($tableName, 'automation')) {
+				$this->addAutomationColumn($tableName);
+			}
+			$row['automation'] = $description;
```
Comment (rskrzypczak) [inline modules/Settings/Picklist/models/Module.php]:
```suggestion
			$row['automation'] = $automation;
```

File: modules/Settings/Picklist/models/Module.php (hunk 2 above)
Comment (konadrian) [reply modules/Settings/Picklist/models/Module.php]: English use of the word as a mass noun, as in considering their relative status.' So status and statuses can both be used as the plural, depending on the context, but status is more common

**Pull request merged**